src/api.js

```diff
@@ -0,0 +1,6 @@
+import { getExamAccessToken, store } from './data';
+
+export default async function checkExamAccessToken() {
```

Comment:
This is the call that will happen in the Unit. Frontend-app-learning is where I'll check that the unit is part of an exam and then update the url to include the access token.